09 News Fixes/gamedata/scripts/modxml_dart_news.script

```diff
@@ -0,0 +1,60 @@
+local log   = DART.createLogger("NEWS")
+local cache = DART.createCache("NEWS")
+
+function parseDXMLNews(DXML)
+    DXML:iterateChildren(DXML:getRoot(), function(child)
+      if not DXML:isElement(child) then
+        return
+      end
+  
+      local factor = 11
+      local name  = DXML:getElementName(child)
+      local attrs = DXML:getElementAttr(child)
+  
+      -- log("DXML MATCH " .. name)
+      -- log("DXML ATTRS " .. attrs.x .. " " .. attrs.y .. " " .. attrs.width .. " " .. attrs.height)
+      -- log("New y " .. attrs.y - DART.hud_health_system_y)
+
+      DART.editAttributes(DXML, child, function(attrs)
+        local y = DART.hud_health_system_y + (factor * DART.getAspectRatio() * DART.HEALTH_HUD.scale)
+        if child.name == "icon_static" then
+          return {
+            x = attrs.x + DART.HUD_SAFE_ZONE,
+            y = attrs.y - y,
+            width = attrs.height
```

Comment:
This someimtes works, haven't found out why. Need to remove XML and see if `width` is set correclty